README.md

```diff
@@ -105,6 +105,30 @@ folder structure as per above and modify the buildspec.yml file to specify the v
                        --py_versions py3
     ```
 
+### Building your image using Docker CLI
+
+To build individual dockerfiles with a much more granular look at layer builds, you can use Docker CLI commands to build DLC
+images by following these steps:
```

Comment:
Nit: we're building images, not dockerfiles here - also there are other use cases than getting a granular look at layers. I'd change this sentence to the following:

```suggestion
If you want to build your docker image directly with Docker CLI instead of using our build scripts, follow these steps below:
```